event-bus/distributed/src/main/java/org/apache/james/events/KeyReconnectionHandler.java

```diff
@@ -56,7 +55,7 @@ public KeyReconnectionHandler(NamingStrategy namingStrategy, EventBusId eventBus
     public Publisher<Void> handleReconnection(Connection connection) {
         return Mono.fromRunnable(() -> {
             try (Channel channel = connection.createChannel()) {
-                QueueArguments.Builder builder = configuration.workQueueArgumentsBuilder(!ALLOW_QUORUM);
+                QueueArguments.Builder builder = configuration.workQueueArgumentsBuilder();
                 configuration.getQueueTTL().ifPresent(builder::queueTTL);
```

Comment:
Are those options compatible?

Reply:
Seems not, good catch

Reply:
Indeed TTL seems to work with quorum queue cf: https://www.rabbitmq.com/docs/quorum-queues#queue-and-per-message-ttl-since-rabbitmq-310